### PR TITLE
close #118

### LINKED ISSUE #118: Adjust spacing / size on team page
Now that we're smaller, we should update this; this was designed for a largish group and looks poor now: 

image

### PR DESCRIPTION
**Pull request by haihan-lin**
### Does this PR close any open issues?
Closes #118

### Give a longer description of what this PR addresses and why it's needed
increase icon sizes and margin

### Provide pictures/videos of the behavior before and after these changes (optional)
Screen Shot 2022-09-19 at 1 50 50 PM

### CODE REVIEW

File: _includes/team_header.html [hunk 1]
```diff
@@ -1,54 +1,68 @@
...
-    <img alt="{{ person.first_name }} {{ person.last_name }}" class="person-img person-image-small offset{{iter}}" title="{{ person.first_name }}  {{ person.last_name }}" src="http://www.centonovelosgatos.com/wp-content/uploads/2013/08/unknown-female.jpg" onmouseenter="highlightPerson('{{ person.key }}')" onmouseleave="clearClass('highlighted')">
+  <img alt="{{ person.first_name }} {{ person.last_name }}" class="person-img person-image-small offset{{iter}}"
+    title="{{ person.first_name }}  {{ person.last_name }}"
+    src="http://www.centonovelosgatos.com/wp-content/uploads/2013/08/unknown-female.jpg"
+    onmouseenter="highlightPerson('{{ person.key }}')" onmouseleave="clearClass('highlighted')">
   {% endif %}
-  </a>
-  {% assign iter = iter | plus: 1 | modulo: 2 %}
+</a>
+{% assign iter = iter | plus: 1 | modulo: 2 %}
```
Comment (JackWilb) [inline _includes/team_header.html:35]: Is everything above this a change to formatting?

File: _includes/team_header.html [hunk 2]
```diff
@@ -1,54 +1,68 @@
...
+      let rColor = Math.floor(Math.random() * 4);
+      people[i].className = `${people[i].className} color${rColor} ${i > 0 ? 'not-first' : ''}`;
+
       // random rotation
-      var rRotation = Math.floor(Math.random() * 21) - 10;
-      rTransform += "rotate(" + rRotation + "deg)";
-      
+      // const rRotation = Math.floor(Math.random() * 21) - 10;
+      // rTransform += "rotate(" + rRotation + "deg)";
+
```
Comment (JackWilb) [inline _includes/team_header.html:58]: Remove?

File: _includes/team_header.html [hunk 3]
```diff
@@ -1,54 +1,68 @@
...
+      const rTranslationY = Math.floor(Math.random() * 11);
+      rTransform += ` translate(${rTranslationX}px,${rTranslationY}px)`;
+
       // update random transformations
-     people[i].style.transform = rTransform;
+      people[i].style.transform = rTransform;
     }
   })();
-</script>
+</script>
```
Comment (JackWilb) [inline _includes/team_header.html:68]: Please make sure we have new lines at the end of the files

**Review by JackWilb — APPROVED** [review]
Looks great. I have a couple minor comments

File: _includes/team_header.html (hunk 1 above)
Comment (haihan-lin) [reply _includes/team_header.html:35]: yes formatting

File: _includes/team_header.html (hunk 2 above)
Comment (haihan-lin) [reply _includes/team_header.html:58]: just in case we want to rotate icons in the future :)

Comment (JackWilb) [reply _includes/team_header.html:58]: Okay, I don't have a strong opinion that it should be removed. We can leave it for now

**Pull request merged**